packages/calcite-components/src/utils/form.tsx

```diff
@@ -498,16 +498,16 @@ function syncHiddenFormInput(component: FormComponent): void {
     let input = extra.pop();
 
     if (!input) {
-      input = ownerDocument.createElement("input");
+      input = ownerDocument?.createElement("input");
```

Comment:
For this and similar checks, should we return early by checking `isServer` instead? In this particular block, the next line will cause an error if `ownerDocument` is `undefined`.